main.cpp

```diff
@@ -213,11 +202,9 @@ bool Computer::runQuickbooks(bool isQBInstalled)
     {
         return false;
     }
-    else
-    {
         return true;
-    }
 }
```

Comment:
Just because you delete the `else { }` doesn't mean you can ignore the formatting and not fix the indentation.
- [ ] revise